src/main/java/org/folio/consortia/domain/converter/SharingInstanceConverter.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.consortia.domain.converter;
+
+import java.util.Objects;
+
+import org.folio.consortia.domain.dto.Metadata;
+import org.folio.consortia.domain.dto.SharingInstance;
+import org.folio.consortia.domain.entity.SharingInstanceEntity;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SharingInstanceConverter implements Converter<SharingInstanceEntity, SharingInstance> {
+
+  @Override
+  public SharingInstance convert(SharingInstanceEntity source) {
+    SharingInstance sharingInstance = new SharingInstance();
+    sharingInstance.setId(source.getId());
+    sharingInstance.setInstanceIdentifier(source.getId());
```

Comment:
Should not we set 'instanceId' to 'instanceIdentifier', or is current one intended conversion?

Reply:
yes, it was mistake. fixed